Tropos/Specs/Controllers/TRApplicationControllerSpec.m

```diff
@@ -98,7 +98,11 @@ void inTimeZone(NSTimeZone *timeZone, void (^callback)(void)) {
                 [applicationController subscribeToNotificationsWithDeviceToken:deviceToken];
             });
 
-            OCMVerify([courier unsubscribeFromChannel:expectedChannel]);
+            XCTestExpectation *expectation = [self expectationWithDescription:@"Unsubscribed from channel"];
+            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 1 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
This hard-coded time delay doesn't seem like the right approach to me, as it's likely to be non-deterministic. In Kiwi you can use `stub:withBlock:` to inject a fake implementation that could be used for things like fulfilling asynchronous expectations. Does OCMock have something similar?